src/components/PortfolioModal.tsx

```diff
@@ -43,6 +45,11 @@ function PortfolioModal({
           />
           <h3>{fellow.name}</h3>
           <p>{fellow.description}</p>
+          <p>
+            <MDXProvider>
```

Comment:
There will be a typescript warning for this saying that components props is compulsory. I believe the types from mdx-js__types is incorrect here, so I ignored it.

Reply:
After reading the [documentation](https://www.gatsbyjs.org/packages/gatsby-plugin-mdx/#shortcodes), it looks like you're only supposed to use this component if you have custom components you want to use (shorthands as per the link).

We can fix it by just removing the MDXProvider, since its not doing anything right now. Or if you have some shorthands in mind, we can make those and just add them to the components prop and it'll all be fixed!

Reply:
ohh, no wonder!